dart/lib/src/hub.dart

```diff
@@ -162,16 +160,14 @@ class Hub {
         'captureMessage called with null parameter.',
       );
     } else {
-      final item = _stack.first;
+      final item = _peek();
       if (item != null) {
         try {
-          // TODO pass the scope
-          sentryId = await item.client.captureMessage(
-            message,
-            level: level,
-            template: template,
-            params: params,
-          );
+          sentryId = await item.client.captureMessage(message,
+              level: level,
+              template: template,
+              params: params,
+              scope: item.scope);
```

Comment:
```suggestion
          sentryId = await item.client.captureMessage(message,
              level: level,
              template: template,
              params: params,
              scope: item.scope,
          );
```

here we should typically use a trailing comma